src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -226,7 +229,7 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 		return false
 	}
 
-	private func generatePPACAndSubmitData(completion: ((Result<Void, PPASError>) -> Void)? = nil) {
+	private func generatePPACAndSubmitData(disableExposureWindowsProbability: Bool = false, completion: ((Result<Void, PPASError>) -> Void)? = nil) {
```

Comment:
Do we really want to disable this from a forced submission call from the dev menu?

Reply:
yes unfortunately, because when you force submit the probability flag is still 0 so the windows will never be added to the submission, only if it is a normal submission from outside the dev menu will everything work fine

Reply:
how ?

Reply:
I though the testers could. But this is not blocking, more a question. Everything is fine.